.github/ISSUE_TEMPLATE/rendering-bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Rendering bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**LaTeX code**
+The code of LaTeX you tried to render:
+```latex
+
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen. You can obtain a LaTeX rendering using [QuickLaTeX](https://www.quicklatex.com/).
```

Comment:
👍 for linking to this.

Reply:
Maybe also encourage them to include the image from QuickLaTeX in this section?

I wonder if we should also include some text like "KaTeX's general goal is to render as LaTeX does, so it's important to show how the current behavior compares with LaTeX's."